adabot/circuitpython_libraries.py

```diff
@@ -757,11 +768,18 @@ def validate_in_pypi(repos):
         return [ERROR_NOT_ON_PYPI]
     return []
 
+def output_handler(message="", quiet=False):
+    """Handles message output to prompt/file for print_*() functions."""
+    if output_filename is not None:
+        file_data.append(message)
+    if verbosity and not quiet:
+        print(message)
+
 def print_circuitpython_download_stats():
     """Gather and report analytics on the main CircuitPython repository."""
     response = github.get("/repos/adafruit/circuitpython/releases")
     if not response.ok:
-        print("request failed")
+        output_handler("request failed")
```

Comment:
Worth changing this to `Analytics request failed.` in the same way the others are more descriptive?

Reply:
Good catch! Missed that one somehow...